opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectsExportsContent.jsx

```diff
@@ -33,7 +33,8 @@ const StixCoreObjectsExportsContentComponent = ({
       subscription.unsubscribe();
     };
   });
-  const stixCoreObjectsExportFiles = data?.stixCoreObjectsExportFiles?.edges ?? [];
+  const stixCoreObjectsExportFiles = [...(data?.stixCoreObjectsExportFiles?.edges ?? [])]
+    .sort((f1, f2) => f2?.node?.lastModified?.localeCompare(f1?.node?.lastModified));
```

Comment:
if we sort only in the frontend, we might not get the whole list of exported files (and not get the latest ones). the API should return the ordered list. 
What is strange in the current implementation is that we only get 25 files, without pagination. So if our last export is not in them, we won't see it.

Reply:
Oh yes.. I'll have a look in the backend instead to change this